client/src/EditEntry.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { data } from './data.js';
+
+export default function EditEntry() {
+  const [title, setTitle] = useState('');
```

Comment:
For Edit, when you get to it, these will need to be initialized from the entry being edited.